environment.yml

```diff
@@ -27,6 +27,7 @@ dependencies:
 - psutil
 - pynwb
 - pyqtgraph
+- pyside6
```

Comment:
I wonder if including this in the deps could create issues with opencv, last time I remember it wants to have its own qt installed. That's why I always use `opencv-python-headless` but I'm not sure if opencv windowed stuff still works but just using the existing qt then.

Reply:
Looks like it pulls in qt6-main, so it's at least a closer version; more likely than sticking with qt5-based distros but something to watch out for.

Unfortunately, right now I think some of the code uses opencv's ability to launch a qt-based GUI, I think. I'd need to check to find out where but I'm pretty sure we do it.